content/sensu-go/6.0/reference/events.md

```diff
@@ -578,6 +580,13 @@ required     | false
 type         | Integer greater than 0
 example      | {{< code shell >}}"occurrences_watermark": 1{{< /code >}}
 
+is_silenced  | |
+-------------|------
+description  | If `true`, the event was silenced at the time of processing. Otherwise, `false`. If `true`, the event.Check will also list the silenced entries that match the event in the `silenced` array.
```

Comment:
```suggestion
description  | If `true`, the event was silenced at the time of processing. Otherwise, `false`. If `true`, the event.Check definition will also list the silenced entries that match the event in the `silenced` array.
```